co-simulation/fed/mosaic-infrastructure/src/main/java/org/eclipse/mosaic/fed/infrastructure/ambassador/InfrastructureInstance.java

```diff
@@ -16,17 +16,19 @@
 
 package org.eclipse.mosaic.fed.infrastructure.ambassador;
 
-import org.eclipse.mosaic.lib.geo.CartesianPoint;
-
 import java.io.IOException;
 import java.net.DatagramPacket;
 import java.net.DatagramSocket;
 import java.net.InetAddress;
+import java.util.List;
+
+import org.eclipse.mosaic.lib.geo.CartesianPoint;
+import org.eclipse.mosaic.lib.objects.detector.Detector;
 
 /**
  * InfrastructureInstance class represents a physical instance of an
- * infrastructure node in the simulated environment.
- * It contains information about the infrastructure node such as its ID,
+ * infrastructure instance in the simulated environment.
+ * It contains information about the infrastructure instance such as its ID,
  * location, target address, and ports.
```

Comment:
I can add to this constructor comment but it is not really in the scope of my PR. Also a lot of it is pretty self explanatory like ID or location or address.

Reply:
https://github.com/usdot-fhwa-stol/cdasim/blob/522233f0ffe87617f00a686883dcf8cd153b7c2f/co-simulation/fed/mosaic-infrastructure/src/main/java/org/eclipse/mosaic/fed/infrastructure/ambassador/InfrastructureInstance.java#L48C1-L49C1
https://github.com/usdot-fhwa-stol/cdasim/blob/522233f0ffe87617f00a686883dcf8cd153b7c2f/co-simulation/fed/mosaic-infrastructure/src/main/java/org/eclipse/mosaic/fed/infrastructure/ambassador/InfrastructureInstance.java#L55

Reply:
In the constructor the parameters are described as follows:
```
    /**
     * Constructor for InfrastructureInstance
     * 
     * @param infrastructureId          the ID of the infrastructure instance.
     * @param targetAddress             the target IP address of the infrastructure instance.
     * @param rxMessagePort             the receive V2X message port of the infrastructure instance.
     * @param timeSyncPort              the receive time synchronization message port of the infrastructure.
     *                         
     * @param simulatedInteractionPort  the receive simulated interaction message port of the infrastructure
     *                                  instance.
     * @param location                  the location of the infrastructure instance in the
     *                                  simulated environment
     */ 
I guess I could add that the location is in the simulation map frame and not a geodetic location, but it seems pretty clear since it says `simulated environment` and the location is a cartesian point and not a geodetic point.
```